Backend/controllers/attendancec.js

```diff
@@ -1,15 +1,43 @@
-const mariadb = require('mariadb');
+const fs = require('fs').promises;
+const pool = require("../helpers/database");
 
-// Create a connection pool
-const pool = mariadb.createPool({
-  host: process.env.DB_HOST,
-  user: process.env.DB_USER,
-  password: process.env.DB_PASS,
-  database: process.env.DB_NAME,
-  connectionLimit: 5
-});
+let tableCreated = false;
+
+const createAttendance = async (req, res) => {
+  const { AttendanceDateTime, StudentID, ScheduleID, IsPresent } = req.body;
+  let connection;
+  try {
+    if (!tableCreated) {
+      const createTableQuery = await fs.readFile('./helpers/attendance.sql', 'utf-8');
+      connection = await pool.getConnection();
+      await connection.query(createTableQuery);
+      tableCreated = true;
+    } else {
+      connection = await pool.getConnection();
+    }
+
+    const result = await connection.query(
+      "INSERT INTO Attendance (AttendanceDateTime, StudentID, ScheduleID, IsPresent) VALUES (?, ?, ?, ?)",
+      [AttendanceDateTime, StudentID, ScheduleID, IsPresent]
```

Comment:
The attributes of attendance table was StudentID, CourseID and Attendance count(INT).... Create the tables and write codes according to the ER diagram. 
Also if you carry a bool IsPresent how will you keep the count of total attendance